LiskHQ/lisky
# Remove circular dependency in utils - Closes #501

### What was the problem?

We had a circular dependency.

### How did I fix it?

I separated log functions out of the print utils file.

### How to test it?

Rebase https://github.com/LiskHQ/lisky/tree/493-config_check on this branch, remove the conditional check here: https://github.com/LiskHQ/lisky/pull/502/files#diff-2beec021ec7bb5dc7eccdf1bdc516465R47 and see if you can run the tests with a corrupted config file (missing required keys).

### Review checklist

* The PR solves #501 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
